src/OrbitGl/CaptureViewElement.h

```diff
@@ -7,6 +7,7 @@
 
 #include "Batcher.h"
 #include "PickingManager.h"
+#include "TimeGraphLayout.h"
```

Comment:
Was that missing?

Reply:
My bad, forgot to erase it. As first I saved it in CaptureViewElement (as @reichlfl is suggesting in the next comment)